src/ui.loader-rendering/configs/default.config.js

```diff
@@ -1,7 +1,7 @@
 export default /*tw*/ {
   wrapper: {
     base: `
-      bg-{color}-600
+      {ULoader} bg-{color}-600
```

Comment:
no it's not a component.

new v-bind should be added.
![image](https://github.com/user-attachments/assets/5a6fd691-3206-4814-85d5-78d78d7b7ef0)